core/coreapi/path.go

```diff
@@ -19,6 +22,12 @@ import (
 // ResolveNode resolves the path `p` using Unixfs resolver, gets and returns the
 // resolved Node.
 func (api *CoreAPI) ResolveNode(ctx context.Context, p path.Path) (ipld.Node, error) {
+	ctx, span := otel.Tracer("").Start(
```

Comment:
feels like you could probably embed the tracer instance inside the CoreAPI struct to save some typing and map lookups

Reply:
yeah I plan on properly plumbing the tracer through before I publish this PR